src/Symfony/Bundle/SecurityBundle/Command/SetAclCommand.php

```diff
@@ -117,7 +117,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        // BC to be removed in 4.0
+        (new SymfonyStyle($input, $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output))->warning('Command "acl:set" is deprecated since version 3.4 and will be removed from SecurityBundle in 4.0. Register symfony/acl-bundle instead.');
```

Comment:
`Register symfony/acl-bundle instead.` -> `Install symfony/acl-bundle to use this command.` ?